lib/content-structure.js

```diff
@@ -0,0 +1,34 @@
+// Define a content type that matches the records we got in the tested space.
+const simplePost = {
```

Comment:
I think we should have these living with the tests instead of abstracting it.

Reply:
They are shared though, and I wanted to set them up in a way that we can get random content for update/create easily

Reply:
We can still keep them here - but refer from the tests ?

Reply:
Maybe they should not be shared. Because probably each entry would be mapped to a specific test. I also prefer to  have no abstraction (even at the expense of duplication) in the tests.

Reply:
:+1: I can move them away